packages/web/hooks/limit-orders/use-place-limit.ts

```diff
@@ -50,7 +51,15 @@ export const usePlaceLimit = ({
   const quoteAsset = swapAssets.toAsset;
   const baseAsset = swapAssets.fromAsset;
 
-  const priceState = useLimitPrice();
+  const tokenIn =
+    orderDirection === OrderDirection.Bid ? quoteAsset : baseAsset;
+  const tokenOut =
+    orderDirection === OrderDirection.Bid ? baseAsset : quoteAsset;
```

Comment:
maybe this is `orderDirection === OrderDirection.Ask`?

Reply:
Personally, with typescript I prefer string literals over enums (in TS I see enums as quite antiquated)

Example:
`type OrderDirection = "bid" | "ask";`

Reply:
Can change this if it's preferred. Gotten used to working in Rust and I use enums quite a lot so probably a bad habit to carry over to TS.